.travis.yml

```diff
@@ -1,12 +0,0 @@
-arch:
-- amd64
-- ppc64le
-sudo: false
-language: python
-python:
-- '3.8'
-- '3.9'
-- '3.10'
-- '3.11'
```

Comment:
If we don't drop this file, we have to maintain these lines which is unnecessary.